labelbox/schema/project.py

```diff
@@ -160,6 +160,35 @@ def labels(self, datasets=None, order_by=None):
         return PaginatedCollection(self.client, query_str, {id_param: self.uid},
                                    ["project", "labels"], Label)
 
+    def export_queued_data_rows(self, timeout_seconds=120):
+        """ Returns all data rows that are currently enqueued for this project.
+
+        Args:
+            timeout_seconds (float): Max waiting time, in seconds.
+        Returns:
+            URL of the data file with this DataRow information. If the server didn't
+            generate during the `timeout_seconds` period, None is returned.
+        """
+        id_param = "projectId"
+        query_str = """mutation GetQueuedDataRowsExportUrlPyApi($%s: ID!)
+            {exportQueuedDataRows(data:{projectId: $%s }) {downloadUrl createdAt status} }
+        """ % (id_param, id_param)
+        sleep_time = 2
+        while True:
+            res = self.client.execute(query_str, {id_param: self.uid})
+            res = res["exportQueuedDataRows"]
+            if res["status"] == "COMPLETE":
+                return res["downloadUrl"]
```

Comment:
Should we raise an exception if status == FAILED?

Reply:
Yup. Added.